#!/usr/bin/env python
PACKAGE = "ltl_automaton_msgs"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("replan_on_unplanned_move", bool_t,  0, "Toggles replanning when received TS state in not the expected next one in the plan", True)
gen.add("check_timestamp", bool_t, 0, "Toggles accepting or discarding received TS state message if timestamp is same as previously received one", True)

exit(gen.generate(PACKAGE, "ltl_automaton_msgs", "LTLAutomatonDP"))